package.json

```diff
@@ -14,10 +14,13 @@
     "axios": "^1.3.4",
     "bootstrap": "^5.2.3",
     "localforage": "^1.10.0",
+    "lodash": "^4.17.21",
```

Comment:
you don't need to use the whole `lodash` library if only you use debounce method from it